### PR TITLE
Feature/design

### PR DESCRIPTION
**Pull request by pnoorossana**
Fixed the guest modals and navbar.

### CODE REVIEW

File: eagleEvents/templates/layout.html.j2 [hunk 1]
```diff
@@ -8,36 +8,43 @@
...
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>                        
+      </button>
+      <a class="navbar-brand" href="event.html.j2">Eagle Event Planner</a>
+    </div>
+    <div class="collapse navbar-collapse" id="myNavbar">
+      <ul class="nav navbar-nav">
+        <li class="active"><a href="/listEvents">Events</a></li>
+        <li><a href="/listCustomers">Customers</a></li>
+        <li><a href="event-planner.html.j2">Event Planners</a></li>
```
Comment (simpleman19) [inline eagleEvents/templates/layout.html.j2]: This should be the corresponding route for event planners

File: eagleEvents/templates/layout.html.j2 [hunk 2]
```diff
@@ -8,36 +8,43 @@
...
+
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>                        
+      </button>
+      <a class="navbar-brand" href="event.html.j2">Eagle Event Planner</a>
```
Comment (simpleman19) [inline eagleEvents/templates/layout.html.j2]: This should link to the corresponding route for list events

File: eagleEvents/templates/layout.html.j2 (hunk 1 above)
Comment (jake6642) [reply eagleEvents/templates/layout.html.j2]: shouldn't this just be like this?
              `<ul class="nav navbar-nav">`
               ` <li><a href="/listEvents">Events</a></li>`
                `<li><a href="/listCustomers">Customers</a></li>`
               ` <li><a href="/listEventPlanners">Event Planners</a></li>`
              `</ul>`

**Comment by pnoorossana** [comment]
Fixed the list events's link.

**Pull request merged**